sys/opencrypto/cryptodev.c

```diff
@@ -716,13 +716,12 @@ cryptodev_op(
 	cse->uio.uio_segflg = UIO_SYSSPACE;
 	cse->uio.uio_rw = UIO_WRITE;
 	cse->uio.uio_td = td;
-	cse->uio.uio_iov[0].iov_len = cop->len;
 	if (cse->thash) {
-		cse->uio.uio_iov[0].iov_len += cse->thash->hashsize;
 		cse->uio.uio_resid += cse->thash->hashsize;
 	}
-	cse->uio.uio_iov[0].iov_base = malloc(cse->uio.uio_iov[0].iov_len,
```

Comment:
These functions in this file have also changed upstream, though I think you would have to patch only one function now (cod_alloc).